src/simulation.hpp

```diff
@@ -165,6 +165,8 @@ template <typename problem_t> class AMRSimulation : public amrex::AmrCore
 	amrex::Real reltolPoisson_ = 1.0e-5;	    // default
 	amrex::Real abstolPoisson_ = 1.0e-5;	    // default (scaled by minimum RHS value)
 	int doPoissonSolve_ = 0;		    // 1 == self-gravity enabled, 0 == disabled
+	amrex::Vector<amrex::MultiFab> phi;
+	amrex::Vector<amrex::MultiFab> rhs;
```

Comment:
This is not needed and will cause excessive memory usage. Can you move rhs back to a local variable?